xcm/pallet-xcm/src/lib.rs

```diff
@@ -153,21 +154,28 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			dest: MultiLocation,
 			beneficiary: MultiLocation,
-			assets: Vec<MultiAsset>,
+			assets: MultiAssets,
+			fee_asset_item: u32,
```

Comment:
This is a `transaction_version` breaking change. Maybe need some additional label so frontend developers can be aware about this?

Reply:
done

Reply:
I think polkadot.js would need updates to include new types and type changes introduced in this PR. cc @jacogr